validphys2/src/validphys/plotoptions/kintransforms.py

```diff
@@ -69,6 +69,31 @@ def xq2map(self, k1:np.array,k2:np.array,k3:np.array,**extra_labels) -> (np.arra
 import numpy as np
 
 
+# TODO
+class identity:
```

Comment:
```suggestion
class Identity:
```
And the corresponding change in all the yaml files

Reply:
The convention in this module is that the classes that are to be used directly by the commondata are to be written in lowercase.

btw, once the process options is completed (and all commondata get actually process labels instead of k1, k2, k3...) this whole module will be removed. So probably we are keeping it forever.

Reply:
>  So probably we are keeping it forever.

Seems about right